source/js/table.bindings.js

```diff
@@ -0,0 +1,43 @@
+/* global ResponsiveTable, MobileTable */
+
+/**
+ * @file
+* Table component binding.
+ */
+(function (Drupal) {
+  'use strict';
+
+  Drupal.behaviors.gentBaseDefineTables = {
+    attach: function (context, settings) {
+
+      var tables = document.querySelectorAll('table');
+      for (var item = 0; item < tables.length; item++) {
+        var tableItem = tables[item];
+
+        // Create wrapper div and wrap it around each table
+        var tableWrapper = document.createElement('div');
+        tableWrapper.classList.add('responsive-table');
+
+        tableItem.parentNode.insertBefore(tableWrapper, tableItem);
+        tableWrapper.appendChild(tableItem);
```

Comment:
There's no need to loop over the tables 3 times. You can wrap all this into 1 `for` loop, where you:

- Add the wrapper around the `table` element.
- Create the `ResponsiveTable` object.
- Create the `MobileTable` object.

Reply:
Fixed.